src/main/java/org/elasticsearch/search/query/QuerySearchResult.java

```diff
@@ -114,6 +120,14 @@ public void aggregations(InternalAggregations aggregations) {
         this.aggregations = aggregations;
     }
 
+    public List<SiblingReducer> reducers() {
```

Comment:
Can it really only work for siblings or is it just that we only have siblings for now?